.github/workflows/ci.yml

```diff
@@ -346,3 +346,42 @@ jobs:
         with:
           name: plasma-${{ matrix.platform.str }}-${{ matrix.cfg.str }}
           path: build/install
+
+  stubs:
+    needs: [linux, macos, windows]
+    runs-on: ${{ matrix.platform.runner }}
+    name: stubs-${{ matrix.platform.str }}
+    strategy:
+      matrix:
+        platform:
+          - { str: macos-arm64, runner: "macos-14" }
+          - { str: macos-x64, runner: "macos-13" }
+          - { str: linux-x64, runner: "ubuntu-22.04" }
+          - { str: windows-x64, runner: "windows-2022" }
+          - { str: windows-x86, runner: "windows-2022" }
```

Comment:
Is there actual value in running this 5 times in CI? Do we have Python APIs that are different between the platforms?

It's a good test that the tool works, but seems excessive for validating the stub files

Reply:
We already had case where an enum constant accidentally had a different value in 32-bit and 64-bit builds, which was uncovered by the stub generator producing different output when run as 32-bit vs. 64-bit. That's what @Hoikas fixed in 254dcb1f07bc9d79d7abcc4fa16745fc1b80852c in this PR and what I discussed in #1781. For cases like that, it's useful to verify the stubs in multiple different environments - though hopefully such discrepancies won't happen often.

Reply:
Basically what @dgelessus says. I know it makes the matrix more complicated, but it's pretty important (IMO) that the stubs be consistent on all of our supported platforms. It would be a bummer for CI to reject a set of stubs that someone literally just generated and committed on their own machine. Each stub job only takes like 30 seconds, so I felt like it's worth the trouble to get that extra bit of assurance.